Classes/Controller/CalendarController.php

```diff
@@ -190,15 +190,18 @@ public function yearAction($year = null)
      *
      * @param int $year
      * @param int $month
+     * @param int $day
      *
      * @return void
      */
-    public function monthAction($year = null, $month = null)
+    public function monthAction($year = null, $month = null, $day = null)
     {
         $date = DateTimeUtility::normalizeDateTime(1, $month, $year);
```

Comment:
Please use the "$date" for that handling and do not create a new $today variable:
$date = DateTimeUtility::normalizeDateTime($day, $month, $year);